modules/rtcpsummary/rtcpsummary.c

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file rtcpsummary.c RTCP summary module
+ * Output RTCP stats at the end of a call if there are any
+ */
+#include <re.h>
+#include <baresip.h>
+
+static void ua_event_handler(struct ua *ua,
+			     enum ua_event ev,
+			     struct call *call,
+			     const char *prm,
+			     void *arg ) {
+	(void)call;
+	struct le *le;
+	const struct rtcp_stats *rtcp;
+	const struct stream *s;
+
+	switch (ev) {
+
+		case UA_EVENT_CALL_CLOSED:
+			debug("rtcpsummary: CALL_CLOSED");
+			for (	le = call_streaml(call)->head;
```

Comment:
I suggest that you move this whole block into its own function that is called from here.

if will make the code more readable, with less indentation.